apps/rush-lib/src/logic/npm/NpmPackage.ts

```diff
@@ -89,13 +89,13 @@ export class NpmPackage extends BasePackage {
    */
   public static createVirtualTempPackage(packageJsonFilename: string, installFolderName: string): NpmPackage {
     const packageJson: IPackageJson = JsonFile.load(packageJsonFilename);
-    const npmPackage: readPackageTree.PackageNode = {
+    const npmPackage: readPackageTree.Node = {
       children: [],
-      error: undefined,
+      error: null, // eslint-disable-line @rushstack/no-null
       id: 0,
       isLink: false,
       package: packageJson,
-      parent: undefined,
+      parent: null, // eslint-disable-line @rushstack/no-null
```

Comment:
Why is this null?

Reply:
The `readPackageTree.Node.parent` and `readPackageTree.Node.error` types are `readPackageTree.Node | null` and `Error | null` respectively.